### DISCUSSION TITLE
Provide package overview and search

### DISCUSSION THREAD

**Discussion by niclashoyer** [wolfi-dev/os#13577] [Ideas]
Like https://pkgs.alpinelinux.org/packages

https://packages.wolfi.dev is not very browser friendly at the moment 😃

**Comment by kranurag7** [comment]
You can consider using https://apk.dag.dev (Thanks Jon)

**Comment by amouat** [comment]
Also note you can use apk tools e.g. `apk search bla` and `apk list`. One trick I use a lot is to search for the package that supplies a binary e.g. I always forget where ldd is and run `apk search cmd:ldd` (you can even do `apk add cmd:ldd` directly).
